src/modules/monotype.js

```diff
@@ -54,30 +54,34 @@ goog.scope(function () {
     var self = this;
     var projectId = self.configuration_['projectId'];
     var version = self.configuration_['version'];
+    function checkAndLoadIfDownloaded() {
+	  if (loadWindow[Monotype.HOOK + projectId]) {
+	    var mti_fnts = loadWindow[Monotype.HOOK + projectId](),
+	    fonts = [];
 
+	    if (mti_fnts) {
+		  for (var i = 0; i < mti_fnts.length; i++) {
+		    fonts.push(new Font(mti_fnts[i]["fontfamily"]));
+		  }
+	    }
+	    onReady(fonts);
+	  } else {
+	    setTimeout(function () {
+	      checkAndLoadIfDownloaded();
+	    }, 0)
```

Comment:
Ah okay, then I understand the change. Is there any chance we could change the timeout to minimum 50ms or so (it's likely browser will clamp it to something like that anyway)?